src/components/NormalLeftRow/index.jsx

```diff
@@ -17,12 +18,23 @@ function NormalLeftRow({ schema, classes }) {
    *  This enables the left row to have matching number of lines with
    *  the right row and align the lines and heights between the two rows.
    */
-  const nonPaddedKeywords = ['type', 'description', 'name'];
   const blankLinePaddings = [];
+  const skipKeywords = ['type', 'description', 'name'];
+  const keywords = Object.keys(schema).filter(
+    key => !skipKeywords.includes(key)
+  );
 
-  Object.keys(schema).forEach(keyword => {
-    if (!nonPaddedKeywords.includes(keyword)) {
-      blankLinePaddings.push(<br key={keyword} className={classes.line} />);
+  keywords.forEach((keyword, i) => {
+    if (i > 0) {
+      blankLinePaddings.push(
+        <Typography
+          key={`${keyword} line`}
+          component="div"
+          variant="subtitle2"
+          className={classes.line}>
+          <br />
+        </Typography>
```

Comment:
Rather than using `<br  />`, is there a way we can use https://material-ui.com/components/dividers/#divider? This would make it easier in the future to override the styles of that component. Also, I don't think we need `Typography` here.